src/store.js

```diff
@@ -1,20 +1,29 @@
-import { createStore, combineReducers, applyMiddleware, compose } from 'redux';
+import { applyMiddleware, combineReducers, compose, createStore } from 'redux';
 import thunk from 'redux-thunk';
-import wishlists from 'modules/wishlists';
+import userReducer from './reducers/userReducer';
+import wishlistsReducer from './reducers/wishlistsReducer';
 
 /**
  * Redux setup
  */
 
```

Comment:
might be a good step to implement the `redux-persist` so we can whitelist which reducer to persist on go instead of doing it in a batch